File: UniOdonto.Portal/UniOdonto.DAL/Entidad/Medicamentos.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace UniOdonto.DAL.Entidad
{
    public class Medicamentos : BaseEntity
    {
        public string Generico { get; set; }
        public string Comercial { get; set; }
        public string Dosis { get; set; }
        public string Presentacion { get; set; }
        public int Cantidad { get; set; }

        public string Indicaciones { get; set; }
        [ForeignKey("ApplicationUser")]
        public string ApplicationUserId { get; set; }
        public virtual ApplicationUser ApplicationUser { get; set; }
    }
}

File: UniOdonto.Portal/UniOdonto.Portal/Scripts/Pages/Operaciones/Medicamentos.js
function medicamentosCallback(data) {
    $('#Id').val(data.Id);
    $('#Generico').val(data.Codigo);
    $('#Comercial').val(data.Codigo);
    $('#Dosis').val(data.Codigo);
    $('#Presentacion').val(data.Codigo);
    $('#Cantidad').val(data.Descripcion);
    $('#Indicaciones').val(data.Descripcion);
    //$('#Nombre').val(data.Nombre);
    //$('#DuracionCitaId option[value=' + data.DuracionCitaId + "]").prop('selected', true);
}